dist/origin-web-catalogs.js

```diff
@@ -213,6 +218,11 @@ webpackJsonp([ 0, 1 ], [ function(e, t) {
                 label: "Python",
                 tags: [ "python" ],
                 icon: "font-icon icon-python"
+            }, {
+                id: "golang",
+                label: "Go",
+                tags: [ "Golang", "golang", "go" ],
```

Comment:
included Golang with capital G for backwards compatibility with the the old categories